hay-espacio-en-el-taco/dbs-scraper
# Filtering improvements

## What's the purpose of this PR?
* Now it generates an `attributes` object during scraping. This object contains all the unique values of all of the fields in the cards.
* Now the filter options are all dynamic by using the new `attributes` property.
* Added Rarity filter
* Fixes the color while hovering on a selected filter button.

![image](https://user-images.githubusercontent.com/7025727/75924789-2bd2dc80-5e2d-11ea-9a19-56967a7b6e10.png)


## How can this be tested?
* Run this branch.
* Use the new _Rarity_ filter.
* Use the other filters and confirm that they still work as expected.
* While hovering on a selected filter button, confirm the color is red.
